src/networking.c

```diff
@@ -4224,6 +4224,11 @@ void helloCommand(client *c) {
 
     addReplyBulkCString(c, "modules");
     addReplyLoadedModules(c);
+
+    if (sdslen(server.availability_zone) != 0) {
+        addReplyBulkCString(c, "availability_zone");
+        addReplyBulkCBuffer(c, server.availability_zone, sdslen(server.availability_zone));
+    }
```

Comment:
Should we just reply with empty string if AZ information doesn't exist? The client doesn't need to figure out if the server supports this field or not then.

Reply:
That was the original implementation, but I requested that we only include it if non-empty, i.e. the feature is used. This is to keep the HELLO reply small.

A client who wants to support older versions still need to handle the case that this field is missing.

Reply:
Ack.